src/ui/server/controllers/insurance/business/turnover/map-submitted-data/index.ts

```diff
@@ -29,15 +31,13 @@ const mapSubmittedData = (formBody: RequestBody): object => {
     populatedData[PERCENTAGE_TURNOVER] = stripCommas(populatedData[PERCENTAGE_TURNOVER]);
   }
 
-  if (objectHasProperty(populatedData, CURRENCY_CODE)) {
-    if (populatedData[CURRENCY_CODE] === ALTERNATIVE_CURRENCY_CODE) {
-      populatedData[TURNOVER_CURRENCY_CODE] = populatedData[ALTERNATIVE_CURRENCY_CODE];
-    } else {
-      populatedData[TURNOVER_CURRENCY_CODE] = populatedData[CURRENCY_CODE];
-    }
+  // map "currency code" and "alternative currency code" fields.
+  populatedData = mapCurrencyCodeFormData(populatedData);
 
+  // map the resulting "currency code" into a single "Turnover currency code" field
+  if (objectHasProperty(populatedData, CURRENCY_CODE)) {
+    populatedData[TURNOVER_CURRENCY_CODE] = populatedData[CURRENCY_CODE];
     delete populatedData[CURRENCY_CODE];
-    delete populatedData[ALTERNATIVE_CURRENCY_CODE];
```

Comment:
is this not set if you choose an alternative currency?

Reply:
Sorry only just seen this - no it's not, 

For all currency fields, it will always be set to `CURRENCY_CODE`.

So if the "alternative" option is selected, `CURRENCY_CODE` will equal "alternative".

the "alternative" field is not in the actual data it's purely for the UI, so we need to delete it (moved to a new helper function in this PR)